client/src/views/Models/components/MetadataPanel.vue

```diff
@@ -1,32 +1,53 @@
 <template>
-  <div v-if="metadata" class="metadata-pane-container">
-    <div v-for="(item, key) in metadata" :key="key">
-      {{item}}
-    </div>
-    <!-- <div class="metadata-item">
-      <div class="key">Name</div>
-      <div>{{metadata.name}}</div>
-    </div>
-    <div class="metadata-item">
-       <div class="key">Description</div>
-      <div>{{metadata.description}}</div>
-    </div>
-    <div class="metadata-item">
-       <div class="key">Created</div>
-      <div>{{metadata.created}}</div>
-    </div>
-    <div class="metadata-item">
-       <div class="key">Source</div>
-      <div>{{metadata.source}}</div>
-    </div>
-    <div class="metadata-item">
-       <div class="key">Version</div>
-      <div>{{metadata.version}}</div>
-    </div>
-    <div class="metadata-item">
-       <div class="key">Knowledge</div>
-      <div><a class="text-link" :href="metadata.knowledge">{{metadata.knowledge}}</a> </div>
-    </div> -->
+  <div class="metadata-panel">
+    <details v-for="(datum, index) in metadata" :key="index">
+
+      <template v-if="isTypeModel(datum)">
+        <summary :title="datum.uid">Model</summary>
+        <h6>Variables</h6>
```

Comment:
I'm not a big fan of underlined text and to be consistent. What about just using `font-weight: bold` instead for these `h6` titles?

Reply:
I can make the text bold instead of underlined.
And I can replace the `<h6>` for a non-descriptive `<span>`.

In the future 2022 version we have to be careful about semantics and accessibility with all those `<div>` and `<span>` soup.